src/test/kotlin/com/zepben/evolve/database/sqlite/cim/upgrade/changesets/network/ChangeSet55NetworkValidator.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.database.sqlite.cim.upgrade.changesets.network
+
+import com.zepben.evolve.database.paths.DatabaseType
+import com.zepben.evolve.database.sqlite.cim.upgrade.changesets.ChangeSetValidator
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import java.sql.Statement
+
+object ChangeSet55NetworkValidator : ChangeSetValidator(DatabaseType.NETWORK_MODEL, 55) {
+
+    //
+    // NOTE: In the validators we are only checking the columns that were actually changed.
+    //
+
+    override fun setUpStatements(): List<String> = emptyList()
+
+    override fun populateStatements(): List<String> = listOf(
+        "INSERT INTO usage_points (mrid, name, description, num_diagram_objects, location_mrid, is_virtual, connection_category, rated_power, approved_inverter_capacity, phase_code) VALUES ('mrid', 'name', 'desc', 0, 'l_id', true, 'connection_category', 2, 2, 'AN');",
```

Comment:
suggestion: Recommend that you order these in the same way as your teardown statements (ideally alphabetically) so you can more easily check that you have cleared all the tables you have populated.

Reply:
set up alphabetically and consistent with tear down.